modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx

```diff
@@ -208,6 +209,14 @@ export default function EditObjectFolder({
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [selectedObjectFolder.objectFolderItems?.length]);
 
+	Liferay.on('beforeNavigate', () => {
+		updatePreviousURLParam('objectFolderName', objectFolderName);
+	});
+
+	window.addEventListener('popstate', () => {
+		updatePreviousURLParam('objectFolderName', objectFolderName);
+	});
```

Comment:
hey @danvb112, just wanted to check with you if its okay to this piece of code to be outside a function or a `useEffect` for exemple, won't it be called every single time a change is made on the page? or this was really your intention?

Reply:
he explained to me that this won't affect the behavior, so i will approve the PR!